crates/bevy_ecs/README.md

```diff
@@ -0,0 +1,38 @@
+# Bevy ECS
+
+[![Crates.io](https://img.shields.io/crates/v/bevy_ecs.svg)](https://crates.io/crates/bevy_ecs)
+[![license](https://img.shields.io/badge/license-MIT-blue.svg)](../../LICENSE)
+[![Discord](https://img.shields.io/discord/691052431525675048.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/gMUk5Ph)
+
+## What is Bevy ECS?
+
+This crate contains the Entity Component System (ECS) implementation used in and developed for the game engine [Bevy][bevy]. Even though it was created as part of the game engine, Bevy ECS can be used standalone and in combination with other projects or game engines.
+
+## About
+
+Entity component system is an architectural pattern using composition to provide greater flexibility.
+
+### Main concepts
+
+* Entities are identifiers
+* Components are data structures that can be attached to entities
+* Systems encode a certain behaviour of sets of entities selected based on their components
```

Comment:
Not necessarily - a system could be operating exclusively on resources too, and this isn't particularly rare. Reframing this to say "behavior on world" could be better.